project/Dependencies.scala

```diff
@@ -0,0 +1,9 @@
+object Dependencies {
+  // scalaVersion needs to be kept in sync with travis-ci
+  val Scala210      = "2.10.7"
```

Comment:
Is this needed for scalaJS?
I think `interplay` only had `scala210` for `sbt 0.13`...

Reply:
Removed.